### ISSUE TITLE
Make it easier to identify the content on each folder

### ISSUE THREAD

**Issue by RaphaelDDL** [JK3Y/NPS-Browser-macOS#5]
#### Make it easier to identify the content on each folder by creating a `name.txt` on each game's folder with the game name

Reasoning: Content folder is titled as it's ID. When having multiple of them, becomes tricky to manage, having to recur to NPS and search for the ID (on Google as NPS search is by title only) to know which content is which. 

By having SwiftNPS save a small txt file on the folder's root containing the content name, it'll make a lot easier to manage. Yes, it'll still be required to go on each folder, but this is quite easy with the keyboard if you use list/columns in Finder, which shows file previews as well.

Example:
<img width="615" alt="screen shot 2018-06-15 at 11 44 37 pm" src="https://user-images.githubusercontent.com/1087109/41495112-19a0043c-70f6-11e8-85e5-0aef673ef8f5.png">

**Comment by JK3Y** [comment]
I'll look into this. The App folder and its contents is generated by pkg2zip. If you're looking to store the files, you can save them as a zip file from the preferences window and the filename for those is the actual game name

**Comment by JK3Y** [comment]
Ahh okay that makes sense. I've been thinking about another implementation with a root "NPS Downloads" folder, and inside that you have the decrypted App folder along with named symlinks to the corresponding folder in the App directory. This gives you access to the title names without having to click on a txt file and doesn't alter the pkg2zip output. Thoughts?

**Comment by RaphaelDDL** [comment]
I'm not much fan of symlinks but that seems to be a cool idea.

**Comment by dhondup2** [comment]
If you still want something to make name.txt inside the game directory I made a little java code to do that. I was also looking to organize my game file so I made a code that organizes it for me. What I wanted was similar to yours so I changed some things to make it do what you wanted. I also included my method as well if you are interested. MakeTextFile.class is the one that puts the "name.txt" file in the extracted game folder and MoveToDir.class is the one I use. I included a screenshot of what mine does. You also need the PSV_GAMES.tsv file from nopaystation.com.
[NoPayStation.zip](https://github.com/JK3Y/NPS-Browser-macOS/files/2214320/NoPayStation.zip)

**Comment by JK3Y** [comment]
@dhondup2 Yes, definitely! I was thinking of the same concept that iTunes uses to "manage your library", which i've always loved (and is the main reason i used iTunes when I didn't even have an iPod and still used Windows) because it automatically placed the files in categories of Artist > Album > Song. I wanted to do kind of the same managing ability so I put this together to visualize it and I think it looks pretty good.
<img width="865" alt="screen shot 2018-07-20 at 8 56 20 pm" src="https://user-images.githubusercontent.com/17938577/43031047-7963a868-8c5f-11e8-8661-00b34119ace5.png">

Thoughts?

**Comment by dhondup2** [comment]
That looks like a good idea to sort it but I feel it might be a good idea to put in the region code in the name as well because if someone were to download same game from different regions it might conflict with the shortcut link.

**Comment by JK3Y** [comment]
@dhondup2 ok like "Bit Dungeon Plus [EU]"? That's doable :) With this setup though, the location that you choose in the Preferences will now have to be changed to NPS Download Library location. Maybe it should be defaulted to '~/NPS' or '~/NPS Downloads'?

**Comment by dhondup2** [comment]
@JK3Y  I don't follow what you mean by default the location to NPS Download.

**Comment by dhondup2** [comment]
I see what you mean by that. Yes, that would make it neater and less confusing.

**Comment by RaphaelDDL** [comment]
Maybe the folder we set-up as the Download folder is the one to be used as base. So `app` will stay as is and all other symlinks are siblings of the `app` one, just like in your screenshot. I think that'll be great.

**Comment by JK3Y** [comment]
Commit b3f5ed8 added separate folders for the consoles. One more step closer.
